pkg/providers/cloudstack/cloudstack.go

```diff
@@ -810,7 +802,7 @@ func buildTemplateMapMD(clusterSpec *cluster.Spec, datacenterConfigSpec v1alpha1
 		values["noProxy"] = noProxyList
 	}
 
-	return values
+	return values, nil
```

Comment:
What's the point of adding a new return error arg for `buildTemplateMapCP` and `buildTemplateMapMD` if we're just going to return `nil` every time?